charts/kube-network-policies/templates/daemonset.yaml

```
@@ -0,0 +1,75 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: {{ template "kube-network-policies.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "kube-network-policies.labels" . | nindent 4 }}
+    {{- with .Values.daemonset.labels }}
+    {{- toYaml . | nindent 4 }}
+    {{- end }}
+  {{- with .Values.daemonset.annotations }}
+  annotations:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+spec:
+  selector:
+    matchLabels:
+      {{- include "kube-network-policies.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      labels:
+        {{- include "kube-network-policies.labels" . | nindent 8 }}
+        {{- with .Values.daemonset.labels }}
+        {{- toYaml . | nindent 8 }}
+        {{- end }}
+    spec:
+      hostNetwork: true
+      dnsPolicy: ClusterFirst
+      {{- with .Values.daemonset.nodeSelector }}
+      nodeSelector:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      {{- with .Values.daemonset.tolerations }}
+      tolerations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      serviceAccountName: {{ include "kube-network-policies.serviceAccountName" . }}
+      containers:
+        - name: kube-network-policies
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          args:
+            - /bin/netpol
+            - --hostname-override=$(MY_NODE_NAME)
+            - --v=2
+            {{- if .Values.baselineAdminNetworkPolicy }}
```

```suggestion
            {{- if .Values.adminNetworkPolicy }}
```

I learned this recently, so there is Admin Network Policy, that has two CRD adminNetworkPolicy and baseAdminNetworkPolicy , so, maybe better to name `Values.adminNetworkPolicy`     instead of `Values.baselineAdminNetworkPolicy`   ??

---

You're right.
JFYI, we specify both flags in [install-anp](https://github.com/kubernetes-sigs/kube-network-policies/blob/main/install-anp.yaml#L83C24-L83C88). I could not come up with a generic parameter name that describes both AdminNetworkPolicy and BaselineAdminNetworkPolicy.
Renamed as suggested.